subprojects/packagefiles/sfml/meson.build

```diff
@@ -1,14 +1,15 @@
 project(
   'sfml',
   'cpp',
-  version: '2.6.2',
+  license: 'Zlib',
+  version: '3.0.1',
   default_options: ['cpp_std=c++17', 'wrap_mode=forcefallback'],
```

Comment:
We've been replacing cpp_std in default options with override_options. Would it be possible here or is it too annoying?

Reply:
Done.